src/pages/Profile/Profile.jsx

```diff
@@ -1,12 +1,10 @@
 import React from "react";
```

Comment:
❓ Did you mean to edit the profile page?

Reply:
Yes, I removed that button we talked about the other day